src/infrastructure/config/schema.js

```diff
@@ -128,16 +132,16 @@ const togglesSchema = new SimpleSchema({
 });
 
 const notificationsSchema = new SimpleSchema({
-  connectionString: patterns.redis
+  connectionString: patterns.redis,
 });
 
-
 const accessIdentifiers = new SimpleSchema({
   identifiers: {
     type: Object,
   },
-  'identifiers.service': patterns.uuid,
-  'identifiers.organisation': patterns.uuid,
+  "identifiers.service": patterns.uuid,
+  "identifiers.organisation": patterns.uuid,
+  "identifiers.departmentForEducation": patterns.uuid,
```

Comment:
Is this meant to be added as part of this PR?

Reply:
I added `identifiers.departmentForEducation` to my local config in the previous ticket I was working on, NSA-7979, and needed to add it here to deploy to dev. Should I have taken it out of my local config?

Reply:
@AMcCartneyMTH @JaredAMarsh identifiers.departmentForEducation was added to my local config in the previous support ticket 7979. Not sure if I should have removed it for this ticket or added it in to the schema file as above?

Reply:
This specific change just makes the quotes into double-quotes to fit the ESLint rules I believe. 👍 The values in `main` have single quotes, so this is just modifying that 😄